examples/deferred_processing/automated_deferred_processing/README.md

```diff
@@ -0,0 +1,342 @@
+# Automated Deferred Processing
+
+[Deferred processing](https://docs.pachyderm.com/latest/how-tos/deferred_processing/) 
+is a Pachyderm technique for controlling when data gets processed.
+Deferred processing uses branches to prevent pipelines from triggering on every input commit.
+This example shows how to automate the movement of those branches,
+by using a cron pipeline.
+The Makefile in this example,
+along with the explanations provided in this document,
+should give you a good start on implementing this in your Pachyderm cluster
+with or without access controls activated.
+
+## Prerequisites
+
+Before you can start working on this example, make sure you have the following prerequisites:
+
+You should understand deferred processing by reading the documentation
+and trying the [deferred processing example](../deferred_processing_plus_transactions).
+That example is used extensively here.
+
```

Comment:
Perhaps mention something along the lines of - The pipeline needs to be able to send signals to the Pachyderm cluster about changing branch labels. Since a pipeline is essentially a sandbox with no knowledge of pachyderm cluster directly, we need to give it credentials to interact with pachd.